source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/admin/settings/index.tsx

```diff
@@ -5,14 +5,18 @@ import { withAdminLayout } from '@ui/components/layout/admin';
 
 import { Switch, Route } from 'react-router-dom';
 import OrganizationsRoute from '@ui/routes/admin/settings/organizations';
+import WorkflowDefinitionsRoute from '@ui/routes/admin/settings/workflow-definitions';
 
 export const pathName = '/admin/settings';
+export const workflowDefinitionsPathName = '/admin/settings/workflow-definitions';
```

Comment:
this doesn't need to be exported and should be in the inner component.
if you are not going use pathName remove it